pydra/engine/helpers_state.py

```diff
@@ -1,3 +1,5 @@
+""" additional functions used mostly by the State class """
```

Comment:
Sorry for the late review here:

@djarecka any reason for separating helpers into their own file instead of including them in `state.py`? 
Not necessarily as part of the `State` class, but within the same file.

Reply:
nothing special, the same reason way we have other files with helping functions.